src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -439,6 +445,11 @@ public CmdletAttribute(string verbName, string nounName)
 
             NounName = nounName;
             VerbName = verbName;
+
+            if (VerbName == VerbsCommon.Get)
+            {
+                ConfirmImpact = ConfirmImpact.Low;
+            }
```

Comment:
This would cause inconsistency between cmdlet and advanced function. Say for the below function, I will see the `ConfirmImpact` value to be medium.
```
function Get-Something
{
    [CmdletBinding(SupportShouldProcess = true)]
    param()
    ...
}
```
From the committee review:
> Should also **consider** changing default for Get- cmdlets to low if not specified.

I suggest we don't do this, so all `Get` command will return `None` for `ConfirmImpact` unless the command declares `ConfirmImpact` itself.

Reply:
Yeah, we should probably avoid disparity between cmdlets and advanced functions if we can.

I read the committee's comment there as "we should consider making `Get-` cmdlets return `Low` for `ConfirmImpact` if and only if `SupportsShouldProcess = true` **and** they do not specify a `ConfirmImpact` explicitly.

And that's what this does. Technically it would always set `Get` cmdlets to Low internally, but nothing will ever register it unless `SupportsShouldProcess = true` thanks to the gated getter I added.

That said -- should I revert this to avoid disparity between cmdlets and advanced functions?

Reply:
> there doesn't seem to be a simple way to make advanced functions do the same, as there is no way to know the verb within CmdletBindingAttribute.

I think it is enough to indicate in the documentation that is the default for advanced functions and that it is necessary to explicitly indicate `ConfirmImpact` for Get verb. We could enhance SriptAnalyzer too.

Reply:
Indeed. I think this change makes _sense_... but I'm not entirely sure this is the appropriate time to apply such a change. Of course, in an ideal world, cmdlet authors would just _specify their own_ `ConfirmImpact` in every case, and I would generally think that should be the recommendation anyway.

I'd hope this is just a general catch-all for the occasional circumstance where it's omitted... but it may well have a wider impact than I'm thinking, so I would love to hear from the other team members as well. 🙂

Reply:
@PowerShell/powershell-committee re-reviewed the issue and agrees NOT to change the default

Reply:
@vexx32 given the new comment from committee review https://github.com/PowerShell/PowerShell/issues/8157#issuecomment-454989330, can you please revert the following changes?
```c#
if (VerbName == VerbsCommon.Get)
{
    ConfirmImpact = ConfirmImpact.Low;
}
```